lib/src/solver/report.dart

```diff
@@ -130,6 +130,39 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    var outdatedPackagesCount = 0;
+
+    final names = _result.packages.map((id) => id.name);
+    for (var name in names) {
```

Comment:
`final name`, but why not just iterate over the `_result.packages`

Reply:
Iterating over `_result.packages` now.